src/tool/csv2yaml.cpp

```diff
@@ -3497,3 +3531,74 @@ static bool read_homunculus_expdb(const char* file) {
 	ShowStatus("Done reading '" CL_WHITE "%d" CL_RESET "' entries in '" CL_WHITE "%s" CL_RESET "'.\n", count, file);
 	return true;
 }
+// Copied and adjusted from mob.cpp
+static bool mob_readdb_group(char* str[], int columns, int current) {
+	std::string group_name = str[0];
+	uint16 mob_id = static_cast<uint16>(strtol(str[1], nullptr, 10));
+	bool is_default = mob_id == 0;
+
+	if (group_name.rfind("MOBG_", 0) == 0)
```

Comment:
rfind is the last found entry.
Maybe better use strncasecmp on str[0]
Also probably give an error, if the name does not start with this and do not convert it.